include/sys/boardctl.h

```diff
@@ -201,20 +208,21 @@
 #define BOARDIOC_INIT              _BOARDIOC(0x0001)
 #define BOARDIOC_FINALINIT         _BOARDIOC(0x0002)
 #define BOARDIOC_POWEROFF          _BOARDIOC(0x0003)
-#define BOARDIOC_RESET             _BOARDIOC(0x0004)
```

Comment:
David, I think it is a good practice always to add your BOARDIOC to the end of this list, this way you void breaking people system for those that uses protected or kernel mode and just update the kernel. Modifying this list this way could render application to stop working or even generate dangerous faults.

Reply:
@acassis - Good point, I never considered that we have binary comparability across releases. 

Done, rebased and force pushed.